test/API/endpoints/index.spec.js

```diff
@@ -170,9 +304,23 @@ describe('Cloud Recording Endpoints', async function () {
   //   } catch (err) {
   //     assert.deepEqual(
   //       err.message,
-  //       'pageSize is not a valid number',
+  //       'A request to Zoom wasn\'t formatted properly: pageSize should be a number.',
   //       'Wrong error message thrown'
   //     );
   //   }
   // });
+
+  // it('lists User Recordings with nextPageToken', async function () {
```

Comment:
It's probably worth testing the pagination behavior (might require a second stub API with pagination)

Reply:
I tested it manually during development but yes we might want automation for the future

Reply:
Yeah! :)